view/frontend/web/internals/recommend.css

```diff
@@ -11,7 +11,7 @@
     width: 110px;
     margin: 0 auto;
 }
-.recommend-component {
+#algoliaRecommend .recommend-component {
```

Comment:
👍 

To be absolutely safe, I think we should apply this specificity to _all_ Algolia Recommend related DOM elements.